pkg/sql/executor_ir_test.go

```diff
@@ -115,6 +117,35 @@ USING sqlflow_models.my_xgb_regression_model;
 `
 )
 
+func goodStream(stream chan interface{}) (bool, string) {
```

Comment:
How about put to `pkg/sql/testing.go`, then other places like `main_test.go` can also reuse this function.

Reply:
Done.

Reply:
Done.